examples/viewer.py

```diff
@@ -0,0 +1,1489 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# type: ignore
```

Comment:
can you add a docstring at the top of what this file does?

Reply:
I'm not planning to merge this. Just recording it here for the record and if anyone wants to try the demo locally.

Reply:
oh but this is so useful :)
I would love to have this somewhere to quickly study the episodes myself.